utils/find_actors.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+import ast
+import os
+
+
+def is_direct_actor_def(ast_node):
+    if not isinstance(ast_node, ast.ClassDef):
+        return False
+
+    has_actor_base = False
+    for class_base in ast_node.bases:
+        # We are looking for direct name 'Actor'
+        if not isinstance(class_base, ast.Name):
+            continue
+
+        if class_base.id == 'Actor':
+            has_actor_base = True
+            break
+
+    return has_actor_base
+
+
+def extract_actor_name_from_def(actor_class_def):
+    actor_name = None
+    for child in actor_class_def.body:
+        if not isinstance(child, ast.Assign):  # We are looking for `name = 'name'`
+            continue
+        assignment = child
+        for target in assignment.targets:
```

Comment:
I would probably kill `lhs` and `assignment` variables and just use `child.value.value` and `child` instead

Reply:
Hmm, the problem is that there is a typo and it should be `rhs` and not `lhs` :D

I would like to definitely keep the `rhs` as it provides you with more information about what is actually assigned as actor_name (it is the value of the right-hand side of the assignment).